docs/crds/gadgets/ebpftop.md

```diff
@@ -8,7 +8,7 @@ ebpftop shows cpu time used by ebpf programs.
 The following parameters are supported:
  - interval: Output interval, in seconds. (default 1)
  - max_rows: Maximum rows to print. (default 20)
- - sort_by: The field to sort the results by (node,namespace,pod,container,progid,type,name,pid,runtime,runcount,cumulruntime,cumulruncount,totalruntime,totalRunCount,mapmemory,mapcount). (default -runtime,-runcount)
+ - sort_by: The field to sort the results by (node,namespace,pod,container,progid,type,name,runtime,runcount,cumulruntime,cumulruncount,totalruntime,totalRunCount,mapmemory,mapcount). (default -runtime,-runcount)
```

Comment:
It's not possible anymore to sort by pid, however I don't think it's a big deal.

Reply:
We could add sorting for virtual columns, if we really need it, later on - but that won't help for the pid field in this case anyways.